src/librustc_save_analysis/lib.rs

```diff
@@ -78,15 +78,13 @@ pub struct SaveContext<'l, 'tcx: 'l> {
 }
 
 #[derive(Debug)]
-pub enum Data {
```

Comment:
These `Data` structures should all be `pub` still, they are expected to be used by clients